pom.xml

```diff
@@ -5,6 +5,37 @@
     <groupId>com.ericsson.jenkinsci.hajp</groupId>
     <artifactId>hajp-common</artifactId>
     <version>1.0.8-SNAPSHOT</version>
+    <name>Jenkins High-Availability Common Library</name>
+    <description>This is common library for the Jenkins High Availabilty Plugin</description>
+    <url>https://github.com/danielyinanc/hajp-common</url>
+
+    <licenses>
+      <license>
+        <name>MIT License</name>
+        <url>http://www.opensource.org/licenses/mit-license.php</url>
+      </license>
+    </licenses>
+
+    <developers>
+      <developer>
+        <name>Daniel Yinanc</name>
+        <email>daniel.yinanc@ericsson.com</email>
+        <organization>Ericsson</organization>
+        <organizationUrl>http://www.ericsson.com</organizationUrl>
+      </developer>
+      <developer>
+        <name>Scott Hebert</name>
```

Comment:
This is a very bad idea to put our emails on a public pom... we are essentially asking for spam. 

Let's use our dummy github emails if we have to... otherwise name should be enough.